src/main/java/org/gridsuite/study/server/service/VoltageInitService.java

```diff
@@ -97,7 +97,7 @@ public UUID runVoltageInit(UUID networkUuid, String variantId, UUID parametersUu
         return restTemplate.exchange(voltageInitServerBaseUri + path, HttpMethod.POST, new HttpEntity<>(headers), UUID.class).getBody();
     }
 
-    private String getVoltageInitResultOrStatus(UUID nodeUuid, String suffix) {
+    public String getVoltageInitResultOrStatus(UUID nodeUuid, String suffix) {
```

Comment:
I don't really like this method where we can put any suffix to the URL so I prefer we keep it private. Even the name is quite confusing here.

Reply:
Done